src/tools/stringTools.cpp

```diff
@@ -391,3 +391,9 @@ bool kiwix::startsWith(const std::string& base, const std::string& start)
         && std::equal(start.begin(), start.end(), base.begin());
 }
 
+
+bool kiwix::isRawHtmlMime(const std::string& mime)
+{
+  return startsWith(mime, "text/html")
+        && mime.find(";raw=true") != std::string::npos;
+}
```

Comment:
`stringTools.h/cpp` is not a good home for such a narrow check. I would rather make this a member function of the `Response` class and decouple it from HTML-ness, since the `raw=true` attribute may later be applied to other content types as well. Thus the member function (with reversed logic) will be

```c++
bool Response::contentDecorationAllowed() const
{
    return m_mimeType.find(";raw=true") == std::string::npos;
}
```

Reply:
Yes that's better. thanks.